.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Update Docs
+
+on:
+  pull_request_target:
+    branches:
+      - main
+  pull_request:
+
+env:
+  RAILS_ENV: test
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Label PR
+        uses: actions/labeler@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think maybe we need to use this github app https://github.com/apps/probot-autolabeler so we can avoid running this all the time which costs us 1 minute at minimum, even if it runs in 1 second

Reply:
not sure though

Reply:
or at least we need to make sure that this thing doesn't run unnecessarily (by skipping it)

Reply:
Labeler runs everytime a PR is opened or updated, the checking of docs runs only if the doc label is set

Reply:
are you sure labeler isn't running on every push?

Now that I'm thinking, maybe you can move the autolabeler feature to the `linters` job of the `CI` workflow. That way we don't spend more money unnecessarily, WDYT?

Reply:
Makes sense.
labeler runs on the conditions set on the action, which is whenever the PR is opened and/or receives changes